### PR TITLE
[DEV-1566] Remove extra underscore from Custom Account Download files

### PR DESCRIPTION
**Pull request by tony-sappe**
**Description:**
Minor tweak to agency prefix from CSV files accidentally added an extra underscore to the custom account downloads.

**Technical details:**
N/A

**Requirements for PR merge:**

1. [x] Unit & integration tests updated (N/A)
2. [x] API documentation updated (N/A)
3. [x] Necessary PR reviewers:
    - [x] Backend
4. [x] Matview impact assessment completed (N/A)
5. [x] Frontend impact assessment completed (N/A)
6. [x] Data validation completed
7. [x] Appropriate Operations ticket(s) created (N/A)
8. [x] Jira Ticket [DEV-1566](https://federal-spending-transparency.atlassian.net/browse/DEV-1566):
    - [x] Link to this Pull-Request
    - [x] Before / After data comparison

**Area for explaining above N/A when needed:**
```
No changes to matviews, data, API, or docs
```

### CODE REVIEW

File: usaspending_api/download/filestreaming/download_generation.py [hunk 1]
```diff
@@ -206,7 +206,7 @@ def parse_source(source, columns, download_job, working_dir, piid, assistance_id
         filters = request["filters"]
         if request.get("limit"):
             agency = ""
-        else:
```
Comment (BrianZito) [inline usaspending_api/download/filestreaming/download_generation.py]: why is this change needed?

Comment (tony-sappe) [reply usaspending_api/download/filestreaming/download_generation.py]: https://federal-spending-transparency.atlassian.net/browse/DEV-1566?focusedCommentId=45266&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-45266
